Atomic/trust.py

```diff
@@ -82,13 +87,20 @@ def __init__(self, policy_filename="/etc/containers/policy.json"):
         super(Trust, self).__init__()
         self.policy_filename = os.environ.get('TRUST_POLICY', policy_filename)
         self.atomic_config = util.get_atomic_config()
+        class Args():
+            def __init__(self):
+                self.debug = None
+                self.assumeyes = None
```

Comment:
This can be called from Atomic/pull.py (the auto-discover trust workflow), in which case it fails to pass in global opts. @baude we discussed this. Not sure if there is a more elegant workaround. It works but does seem silly.